### PR TITLE
Hydration Issue Resolved 

### PR DESCRIPTION
**Pull request by itzkushagra**
Text content rendered on the server (SSR - Server-Side Rendering) doesn't match the content rendered on the client side, resulting in a "hydration mismatch" during the React rendering process. In order to resolve the issue I have rectified this error by delaying rendering parts of the component until the theme has been loaded on the client to avoid mismatches between server and client renders.

### CODE REVIEW

File: package.json
Comment (JhohannesK) [inline package.json:1]: since we don't have frontend and backend dir anymore
don't include them

File: package-lock.json
Comment (JhohannesK) [inline package-lock.json:1]: you are not supposed to push your lock file.

**Review by JhohannesK — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (JhohannesK) [reply package-lock.json:1]: You also have some conflicts

Comment (itzkushagra) [reply package-lock.json:1]: can you please guide me how to correct these issue  ?

Comment (itzkushagra) [reply package-lock.json:1]: will work on it

File: package.json
Comment (JhohannesK) [reply package.json:1]: revert your commit.
package.json has been added to the .gitignore now so you just have to check this time around if it's still being tracked or not. (it shouldn't be tracked this time around though)
if it isn't, then you commit again and push.

File: CONTRIBUTING.md
Comment (JhohannesK) [inline CONTRIBUTING.md:1]: Please remove one of the header additions

`Contribute in framework`

Comment (itzkushagra) [reply CONTRIBUTING.md:1]: I am having some difficulty in pushing the changes, I haven't done any changes to Contributing.md or readme but there is conflict there can I create a new PR with latest forked repository ?

Comment (JhohannesK) [reply CONTRIBUTING.md:1]: yeah you can

Comment (JhohannesK) [reply CONTRIBUTING.md:1]: I'm merging this...so you go ahead with the new one.

**Pull request merged**